contrib/IECoreUSD/src/IECoreUSD/USDScene.cpp

```diff
@@ -213,8 +216,38 @@ void writeSetInternal( const pxr::UsdPrim &prim, const pxr::TfToken &name, const
 	collection.CreateIncludesRel().SetTargets( targets );
 }
 
+boost::container::flat_map<IECore::InternedString, pxr::TfType> g_schemaTypeSets = {
+	{ "__cameras", pxr::TfType::Find<pxr::UsdGeomCamera>() },
+	{ "usd:pointInstancers", pxr::TfType::Find<pxr::UsdGeomPointInstancer>() }
```

Comment:
Note the current convention from SceneCache would be `ObjectType:<type>`, should we stick to that naming or leave it as a per-format prefix?

In production we've found those ObjectType sets useful for meshes, curves, and points too. Is it worth adding those to `g_schemaTypeSets` as well?

Reply:
In the point instancer case, the `ObjectType` (if we decided to provide it) would be `PointsPrimitive`, which wouldn't distinguish it from regular points primitives. The intention with the `usd:pointInstancers` set is that it allows a single `GafferScene.Instancer/GafferScene.SetFilter` combo to process all UsdGeomPointInstancers in a USD file without needing any knowledge of their specific locations. Getting that lined up smoothly seems to have been a focus of many initial uses of USD, so it seemed important to facilitate it.

I'd like to give `ObjectType:type` further consideration before enshrining it outside of IE. I suspect (but don't know) Cs have their own convention, but more importantly I'd like to consider if we could support this for all objects in Gaffer, not just those loaded from caches. I suspect that might bring up other considerations.

Reply:
Ok, cool, it'd be great to have for natively authored Gaffer objects as well. Totally happy to change the convetion in SceneCache from `ObjectType:` to something else as well.